lib/personal.js

```diff
@@ -36,7 +38,7 @@ function add(buf) {
       word = word.slice(1)
     }
 
-    self.add(word, model)
+    self.add(word, model, NO_CODES)
```

Comment:
Wait, from the code above, it seems `.add` does not accept codes?

Reply:
The signature of that function should be: `function add(dict, word, codes, options)` so it looks like it does need them? But I'm not too sure honestly, without types and a deeper understanding of how the library works I'm not sure about what each functions needs and does.

Reply:
nspell.add is in the readme, described as taking one or two parameters.

You can test this out locally, by removing it, and then using 'npm test'. While we don't use typescript, we do have tests

Reply:
Right, that extra argument is unnecessary then.

Reply:
Removed.